core/src/main/resources/hudson/PluginManager/check.jelly

```diff
@@ -25,9 +25,9 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
 
   <j:invokeStatic var="ds" className="jenkins.model.DownloadSettings" method="get"/>
-  <form method="post" action="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}">
-    ${%lastUpdated(app.updateCenter.lastUpdatedString)}
+  ${%lastUpdated(app.updateCenter.lastUpdatedString)}
+  <f:link href="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}" post="true">
     <f:submit value="${%Check now}" />
```

Comment:
> If you know which css class to use to get button look'n feel on a link

Out of my depth I am afraid. In the meantime I would suggest

```xml
<f:link href="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}" post="true">${%Check now}</f:link>
```

as the supported and safe technique, even if the UI is not exactly what you want.